tests/core/test_array.py

```diff
@@ -114,3 +134,37 @@ def test_trapz_product():
     s = trapz_product((x1, f1), (x2, f2))
 
     assert np.allclose(s, 1.0)
+
+
+def test_cumtrapz():
```

Comment:
I'd really like some kind of documentation for all new tests we add. Preferably a docstring. Can we also have some comments?